src/main/java/br/edu/ufcg/computacao/alumni/core/MatchesFinder.java

```diff
@@ -55,7 +55,7 @@ public static synchronized MatchesFinder getInstance() {
 
 			String scoreString = String.format("%d", score);
 
-			if (score >= 1) {
+			if (score >= 20) {
```

Comment:
Faz alguma diferença? Não seria mais lógico que isso fosse score > 0?

Reply:
Só serviria pra filtrar ainda mais o mapeamento, mas acho que no momento é inútil mesmo.

Reply:
Acho que esse filtro deve ser aplicado pelo cliente. Essa função retorna qualquer match que tenha um score > 0. O cliente pode filtrar os matches de interesse baseando-se em critérios próprios, considerando o score.

Reply:
Verdade, não tinha pensado nisso.